skyportal/facility_apis/ztf.py

```diff
@@ -407,6 +412,47 @@ def remove(request):
 
         DBSession().add(transaction)
 
+    @staticmethod
+    def retrieve(allocation, start_date, end_date):
+
+        """Retrieve executed observations by ZTF.
+
+        Parameters
+        ----------
+        allocation: skyportal.models.Allocation
+            The allocation with queue information.
+        """
+
+        altdata = allocation.altdata
+        if not altdata:
+            raise ValueError('Missing allocation information.')
+
+        jd_start = Time(start_date, format='datetime').jd
+        jd_end = Time(end_date, format='datetime').jd
+
+        if not jd_start < jd_end:
```

Comment:
Is there any clever reason not to do `jd_start >= jd_end` that has to do with `None` values or something?

Reply:
fixed.